pom.xml

```diff
@@ -24,6 +24,7 @@
         <version.org.thymeleaf>3.0.9.RELEASE</version.org.thymeleaf>
         <version.org.thymeleaf.extras.java8time>3.0.1.RELEASE</version.org.thymeleaf.extras.java8time>
         <version.org.wildfly.swarm>2018.2.0</version.org.wildfly.swarm>
+        <version.apache.commins.lang>3.8.1</version.apache.commins.lang>
```

Comment:
Typo: `commins` -> `common`. Not even mentioning that Apache Commons should generally be treated as obsolete crap.